woocommerce/woocommerce-ios
# [Order Creation] Update product row's `price × quantity` view

## Description

Addresses the PR review here: https://github.com/woocommerce/woocommerce-ios/pull/10895#issuecomment-1755794962

> The × symbol looks huge. It's from another PR but I think it'd be better if you pasted the multiplication symbol into the code and used a localized format string rather than using an image.

## Changes
- Collapse and extract the views that form "price", "x", and "quantity" into an unique variable in the view model.
- Use `String.localizedStringWithFormat` to form the formatted output
- Add Unit Tests

## Testing instructions
* Go to Orders > `+` > `+ Add Products`
* See that the "price x quantity" lines render appropriately:

<img width=800 src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/5e279e2f-6b52-4c2c-bf35-302ad5a242bf">
